packages/rocketchat-2fa/client/accountSecurity.html

```diff
@@ -0,0 +1,57 @@
+<template name="accountSecurity">
+	<section class="page-container page-home page-static">
+		<header class="fixed-title border-component-color">
+			{{> burger}}
+			<h2>
+				<span class="room-title">{{_ "Security"}}</span>
+			</h2>
+		</header>
+		<div class="content">
+			<div class="rocket-form">
+				<fieldset>
+					<div class="section">
+						<h1>{{_ "Two-factor_authentication"}}</h1>
+						<div class="section-content border-component-color">
+							<div class="alert pending-background pending-color pending-border">
+								<strong>
+									WARNING: Once you enable this, you will not be able to login on the native mobile apps (Rocket.Chat+) using your password until they implement the 2FA.
```

Comment:
I believe this was only meant to be temporary until the mobile applications actually support it, which is why it isn't translated and is only in english.